src/hooks/useMainProcess.ts

```diff
@@ -135,7 +135,8 @@ export const useMainProcess = () => {
           setAnchorSessionParams(anchorSessionParams);
 
           const fetchAndUpdateSep24Url = async () => {
-            const firstSep24Response = await sep24First(anchorSessionParams);
+            let firstSep24Response = await sep24First(anchorSessionParams);
+            firstSep24Response.url = firstSep24Response.url + '&callback=postMessage';
```

Comment:
@gianfra-t Probably all anchors will have query parameters in their sep24 url but there is no guarantee that it will be like this (could also be more modern path parameters) and as far as I understand it is no requirement of SEP24.

So it could be better to do this defensively:

```JS
const url = new URL(firstSep24Response.url);
url.searchParams.append("callback", "postMessage");
firstSep24Response.url = url.toString();

```